<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>241</width>
     <height>155</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="1">
     <widget class="QPushButton" name="pushUpdate">
      <property name="text">
       <string>Update Settings</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelPort">
      <property name="text">
       <string>Port:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditIP"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelTopic">
      <property name="text">
       <string>Topic:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditPort"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelIP">
      <property name="text">
       <string>IP Address:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditTopic"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEditFinalForm">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Assembled:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Update Peroid (ms):</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditPeriod"/>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEditDisp">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>331</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditIP</tabstop>
  <tabstop>lineEditPort</tabstop>
  <tabstop>lineEditTopic</tabstop>
  <tabstop>pushUpdate</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushUpdate</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>handleUpdateClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>handleUpdateClicked()</slot>
 </slots>
</ui>
